modules/ROOT/pages/shared-resources.adoc

```diff
@@ -95,14 +98,39 @@ You can configure the <<create_domain_project,domain project>> by defining share
                http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
                http://www.mulesoft.org/schema/mule/ee/domain http://www.mulesoft.org/schema/mule/ee/domain/current/mule-domain-ee.xsd">
 
-    <!-- configure here resource to be shared within the domain -->
+    <!-- Example configuration for an HTTP Listener-->
+    <http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" >
+      <http:listener-connection host="0.0.0.0" port="8081" />
+    </http:listener-config>
 
 </domain:mule-domain>
 ----
+
+=== Configure Shared Resources in the Global Elements Tab
+
+You can configure shared resources by creating connector configurations in the *Global Elements* tab in Anypoint Studio. Before you can create connector configurations using Studio, you must add the module to your project.
+
+For example, to add the HTTP connector to the domain project:
+
+. In the *Global Elements* tab, click *Manage Modules*.
++
+image:mruntime-domain-manage-modules.png[manage modules domain]
+. Click *Add Module*.
++
+image:mruntime-domain-add-module.png[add modules domain]
+. Search for the module you want to add and click *Add >*
 +
-* Global Elements: This view provides a graphical interface for defining shared resources.
+image:mruntime-domain-add-module-finish.png[add module finish domain]
+. Click *Finish*.
 
-You can add module or connector dependencies to you domain project so you can use them as shared resources.
+After you add the connector to the domain project, you can create a global configuration to use as a shared resource:
```

Comment:
Does "after you add the connector to the domain project" refer to the previous steps about adding a module?

Reply:
And does a "global configuration" = "configuration domain"?